src/Command/FetchConferencesCommand.php

```diff
@@ -197,4 +197,18 @@ protected function updateExistingConference(Conference $existingConference, Conf
 
         return $updated;
     }
+
+    private function shouldBeIgnored(Conference $conference): bool
+    {
+        foreach ($this->conferenceFilterRepository->findAll() as $conferenceFilter) {
```

Comment:
This `findAll` call will be repeated many time, as the `shouldBeIgnored` function is called inside 2 foreach loops. May be an initial call that store the result in a variable will be faster and avoir many DB queries?